src/test/java/org/simplify4u/plugins/ValidationChecksumTest.java

```diff
@@ -82,7 +83,7 @@ public void testValidationChecksumBuilderChecksumArtifactsNullFails() {
                 .artifactId("maven-compiler-plugin").packaging("jar").version("1.0").build();
         final Artifact a2 = TestArtifactBuilder.testArtifact().groupId("org.apache.commons").artifactId("commons-io")
                 .packaging("jar").version("1.0").build();
-        assertThatCode(() -> builder.artifacts(asList(a1, a2, null)).build())
+        assertThatThrownBy(() -> builder.artifacts(asList(a1, a2, null)).build())
```

Comment:
in this case `asList(a1, a2, null)` should be assigned to variable, and don't call `build` method

Reply:
That won't throw NPE without the build as the not null check is on the array.

Reply:
I've changed the assertion to check for null.